projects/kserve/testing/test_e2e.py

```diff
@@ -557,7 +557,7 @@ def test_consolidated_model(consolidated_model, namespace=None):
         protos_path=protos_path.absolute(),
         call=llm_config["call"],
         llm_path=llm_config["src_path"],
-        threads=llm_config["threads"],
+        concurrency=llm_config["concurrency"],
```

Comment:
I understand that no proto is required anymore with v2,
could you remove the code generating them in this file, as well as the proto files?
(we'll keep the roles, they might be useful again in the future, who knows)